### PR TITLE
H elev

### PR DESCRIPTION
**Pull request by honglioslo**
change elev_breaks as a par

### CODE REVIEW

File: R/assemble_data.R [hunk 1]
```diff
@@ -35,7 +35,7 @@ get_metadata <- function(regine_main = meta_data$regine_main) {
 #' @return A list with data for each station
 #' @export
 
-load_data_mean <- function(path_met, path_runoff, regine_main, time_vec, file_type) {
+load_data_mean <- function(path_met, path_runoff, regine_main, elev_breaks = seq(-200, 4000, 200), time_vec, file_type) {
```
Comment (jmgnve) [inline R/assemble_data.R]: Can you please move elev_breaks to the end of the function call:

path_met, path_runoff, regine_main, time_vec, file_type, elev_breaks = seq(-200, 4000, 200)

Otherwise my scripts using this function will break.

File: R/assemble_data.R [hunk 2]
```diff
@@ -289,7 +288,7 @@ load_data_elev <- function(path_met, path_runoff, regine_main, time_vec, file_ty
 
     # Aggregate data
 
-    met_data <- mapply(aggregate_elevations, met_data, data_all, SIMPLIFY = FALSE)
+    met_data <- mapply(aggregate_elevations, met_data, data_all, MoreArgs = list(elev_breaks = elev_breaks), SIMPLIFY = TRUE)
```
Comment (jmgnve) [inline R/assemble_data.R]: Why did you change SIMPLIFY from FALSE to TRUE? I think it should be FALSE.

File: R/assemble_data.R (hunk 1 above)
Comment (honglioslo) [inline R/assemble_data.R]: Sorry, this is removed here, since load_data_mean does not require ele

File: R/assemble_data.R (hunk 2 above)
Comment (honglioslo) [inline R/assemble_data.R]: This is done
